src/styles/_body.scss

```diff
@@ -0,0 +1,7 @@
+.body {
+  margin: 0;
```

Comment:
I would recommend you to apply this parameters to your div .class calendar, it will allow you to center your content and  formatting the main content directly. I would only left margin 0 to body.

Reply:
guys I am stuck here. can you help?

Reply:
solved it ! :) thanks